controls/cis_ocp_1_4_0/section-4.yml

```diff
@@ -12,13 +12,16 @@ controls:
     - id: 4.1.1
       title: Ensure that the kubelet service file permissions are set to 644 or more
         restrictive
-      status: pending
-      rules: []
+      status: automated
+      rules:
+        - file_permissions_worker_service
       level: level_1
     - id: 4.1.2
       title: Ensure that the kubelet service file ownership is set to root:root
-      status: pending
-      rules: []
+      status: automated
+      rules:
+        - file_owner_worker_service
+        - file_groupowner_worker_service
       level: level_1
     - id: 4.1.3
       title: If proxy kube proxy configuration file exists ensure permissions are
```

Comment:
Why is file_permissions_proxy_kubeconfig not used here? Because it's SDN only?

Reply:
Same question about `file_owner_proxy_kubeconfig` and `file_groupowner_proxy_kubeconfig`

Reply:
Just missed these because I was looking in the wrong profile. Good catch.